lib/asbind-instance/type-converters.js

```diff
@@ -91,7 +91,10 @@ export const converters = new Map([
   // Technically this matches types that don’ exist (like f8),
   // but since those can only appear if the compiler accepts them,
   // it seems unlikely for that to be a problem.
-  [/^(i|u|f)(8|16|32|64)$/, { ascToJs: nop, jsToAsc: nop }],
+  [
+    /^(i|u|f)(8|16|32|64)|[ui]size|bool|externref$/,
```

Comment:
Someone might declare their own `type f8 = something`. Would that be a problem?

Reply:
No, because I am looking at the resolved type. 

(I actually think you sometimes might want to declare a type alias and associate different handling with it, but I’ll solve that another day)